theme/static/css/pages/features.css

```diff
@@ -1,9 +1,9 @@
 #hardware.cover-bg {
-    background-image: url('/theme/images/cover/hardware-bg.jpg');
+    background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('/theme/images/cover/hardware-bg.jpg');
```

Comment:
yeah, this being slightly darker than the library gradient works because the image has higher contrast.